test/Deserializers/Tests/Phpunit/Exceptions/InvalidAttributeExceptionTest.php

```diff
@@ -3,40 +3,37 @@
 namespace Deserializers\Tests\Phpunit\Exceptions;
 
 use Deserializers\Exceptions\InvalidAttributeException;
+use Exception;
+use PHPUnit_Framework_TestCase;
 
 /**
  * @covers Deserializers\Exceptions\InvalidAttributeException
  *
  * @licence GNU GPL v2+
  * @author Jeroen De Dauw < jeroendedauw@gmail.com >
+ * @author Thiemo Mättig
  */
-class InvalidAttributeExceptionTest extends \PHPUnit_Framework_TestCase {
+class InvalidAttributeExceptionTest extends PHPUnit_Framework_TestCase {
 
 	public function testConstructorWithOnlyRequiredArguments() {
-		$attributeName = 'theGame';
-		$attributeValue = 'youJustLostIt';
```

Comment:
![all-work-and-no-play](https://cloud.githubusercontent.com/assets/146040/11184134/35a48b10-8c76-11e5-8a0e-593f8d15ba94.jpg)